redis-plugin/src/main/java/org/entando/entando/plugins/jpredis/aps/system/redis/DefaultCacheConfig.java

```diff
@@ -0,0 +1,48 @@
+package org.entando.entando.plugins.jpredis.aps.system.redis;
+
+import java.util.Collection;
+import java.util.List;
+import org.entando.entando.aps.system.services.cache.ExternalCachesContainer;
+import org.entando.entando.plugins.jpredis.aps.system.redis.condition.RedisActive;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.cache.Cache;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.CachingConfigurerSupport;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+@EnableCaching
+@RedisActive(false)
+public class DefaultCacheConfig extends CachingConfigurerSupport {
+
+    private static final Logger logger = LoggerFactory.getLogger(DefaultCacheConfig.class);
+
+    @Autowired(required = false)
+    private List<ExternalCachesContainer> defaultExternalCachesContainers;
+
+    @Primary
+    @Bean
+    @Autowired
+    public CacheManager cacheManager(@Qualifier(value = "entandoDefaultCaches") Collection<Cache> defaultCaches) {
+        logger.warn("** Redis not active **");
+        DefaultEntandoCacheManager defaultCacheManager = new DefaultEntandoCacheManager();
+        defaultCacheManager.setCaches(defaultCaches);
+        defaultCacheManager.setExternalCachesContainers(this.getDefaultExternalCachesContainers());
+        defaultCacheManager.afterPropertiesSet();
+        return defaultCacheManager;
+    }
+
+    protected List<ExternalCachesContainer> getDefaultExternalCachesContainers() {
```

Comment:
Why protected ? @Configuration class shouldn't be extended ... and here we are propagatin a possible null ... so i would like return Optional to make it know....

Reply:
I've removed the entire method and also the setter, since they are not necessary.

Reply:
ok